### ISSUE TITLE
appsignal_instrument_method produces warnings for methods with kwargs on Ruby 2.7

### ISSUE THREAD

**Issue by grk** [appsignal/appsignal-ruby#656]
## Describe the bug

When a method is using keyword arguments, using appsignal_instrument_method on it produces warnings on Ruby 2.7: 
```
/Users/grk/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/appsignal-2.11.0/lib/appsignal/integrations/object.rb:28: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
```

## To Reproduce

- Using AppSignal for Ruby gem version 2.11.0
- Using Ruby 2.7.1
- With this code:
  ```ruby
  def some_method(foo:, :bar)
  end
  appsignal_instrument_method :some_method
  ```

**Comment by shairyar** [comment]
Hi @grk thanks for bringing it to our notice, we will look into it and once resolved will post an update.

**Comment by tombruijn** [comment]
Looks like the double splat `**` that we need, or is suggested, is not available on Ruby 1.9. So we can pick this up after #676 is merged, where Ruby 1.9 support is dropped. So it will be part of the AppSignal for Ruby gem 3.0 series.

**Comment by tombruijn** [comment]
@grk A fix for this has been released as ruby gem version 2.11.4 🎉

**Comment by grk** [comment]
Awesome, thank you!
